user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/GQI_DxM.md

```diff
@@ -0,0 +1,87 @@
+---
+uid: GQI_DxM
+---
+
+# GQI DxM
+
+The GQI DxM is a [DataMiner Extension Module (DxM)](xref:DataMinerExtensionModules) that hosts the GQI core engine, enabling efficient data retrieval through [GQI queries](xref:About_GQI).
+
+Running GQI as an extension module has several advantages:
+
+- **Independent updates**: Using a [web-only upgrade package](xref:Upgrading_Downgrading_Webapps), GQI can be updated independently from the main DataMiner core software, eliminating the need for a core software restart during updates.
+- **Load balancing**: With the DxM, GQI operations can distribute system load more effectively across multiple DataMiner Agents within a DataMiner System (DMS).
+- **Resource visibility**: Enhanced monitoring capabilities provide greater insight into the resource consumption of individual extensions.
```

Comment:
It isn't clear to me here what the "enhanced monitoring capabilities" are. Do you mean that it's easier to monitor the exact resource consumption of GQI when it's a separate extension?

Reply:
Not as GQI as a whole. GQI supports running your own extensions. Each extension will now run within its own process. This is visible in task manager, which gives more visibility which extension is taking most resources.